Audiobook Player/PlayerViewController.swift

```diff
@@ -148,7 +175,7 @@ class PlayerViewController: UIViewController {
         
         MBProgressHUD.showAdded(to: self.view, animated: true)
         //replace player with new one
-        PlayerManager.sharedInstance.load(self.currentBook) { (audioPlayer) in
+        PlayerManager.sharedInstance.load(self.currentBooks) { (audioPlayer) in
```

Comment:
See the final review comments about the new proposed flow to see how this could be changed